src/component/book.js

```diff
@@ -1,23 +1,31 @@
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { removeBook, fetchBooks } from '../redux/books/books';
 import './books.css';
 
 const Book = () => {
-  const books = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.bookStore);
   const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
```

Comment:
- [ ] You should be able to render books added or deleted with the need to refresh the page. Right now I have to refresh the page to see a newly created book display in the page. It is also the same for remove. Therefore, you need to add another useEffect that depends on the list books and triggers page rerender if modified.
```
  useEffect(() => {
   setAllBooks(books);
  }, [books]);
```
### After Delete before refresh
![image](https://user-images.githubusercontent.com/44057863/180569740-b0233f9d-f58f-48ac-8bd9-7382255c251a.png)
### After Refresh
![image](https://user-images.githubusercontent.com/44057863/180569779-3c409bd9-0dbc-4d36-ab24-1880bdbbfa9c.png)